dali/pipeline/operators/decoder/nvjpeg_decoder_new_test.cc

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/test/dali_test_decoder.h"
+
+namespace dali {
+
+template <typename ImgType>
+class nvjpegDecodeNewTest : public GenericDecoderTest<ImgType> {
+ protected:
+  const OpSpec DecodingOp() const override {
+    return OpSpec("nvJPEGDecoderNew")
+      .AddArg("device", "mixed")
+      .AddArg("output_type", this->img_type_)
+      .AddArg("hybrid_huffman_threshold", hybrid_huffman_threshold_)
+      .AddInput("encoded", "cpu")
+      .AddOutput("decoded", "gpu");
+  }
+
+  void JpegTestDecode(int num_threads, unsigned int hybrid_huffman_threshold = 450*450) {
+    hybrid_huffman_threshold_ = hybrid_huffman_threshold;
+    this->SetNumThreads(num_threads);
+    this->RunTestDecode(t_jpegImgType, 0.7);
```

Comment:
is this passing? In my case I had to change it to 0.9 or so

Reply:
It passes in our runner/devel docker, on my system